crates/iota-json-rpc/src/error.rs

```diff
@@ -526,7 +604,7 @@ mod tests {
             let expected_code = expect!["-32002"];
             expected_code.assert_eq(&error_object.code().to_string());
             let expected_message = expect![
-                "Transaction execution failed due to issues with transaction inputs, please review the errors and try again: Could not find the referenced object 0x0000000000000000000000000000000000000000000000000000000000000000 at version None."
+                "Transaction execution failed due to issues with transaction inputs, please review the errors and try again:\n- Could not find the referenced object 0x0000000000000000000000000000000000000000000000000000000000000000 at version None"
```

Comment:
Nit: Same as above. You can use [TransactionDigest](https://iotaledger.github.io/iota/iota_types/digests/struct.TransactionDigest.html) methods for transparency of the derivation of the values. Will introduce better readability and allow the code to be more easily maintained.

Reply:
Unfortunately I can't use the expect macro with that approach, because it expects a `&'static str`